packages/fxa-auth-server/lib/senders/emails/templates/verifyLogin/index.txt

```diff
@@ -0,0 +1,11 @@
+verifyLogin-title = "Activate the Firefox family of products"
+
+verifyLogin-description = "For added security, please confirm this sign-in from the following device:"
+
+<%- include('/partials/location/index.txt') %>
+
+verifyLogin-action = "Confirm sign-in:"
+<%- link %>
+
+<%- include('/partials/changePassword/index.txt') %>
+<%- include('/partials/support/index.txt') %>
```

Comment:
This doesn't appear to be in the original plaintext version. I'm also okay with keeping it here since it makes sense in this context, but should we consult with product folks for additions like this?

Reply:
Heh yeah, good catch. I also noticed it was missing in the old version, but that seems like a mistake to me. We include this partial for any other time we have that support text in the HTML.

How about I leave it, and start a doc of some sort to keep note of anything like this that we can run by product or design or whomever to confirm?

Reply:
That seems reasonable to me!